src/v2/Apps/Categories/__tests__/CategoriesApp.jest.tsx

```diff
@@ -30,7 +27,7 @@ const { getWrapper } = setupTestWrapper<CategoriesApp_Test_Query>({
 
 describe("CategoriesApp", () => {
   it("renders", () => {
-    const wrapper = getWrapper()
-    expect(wrapper.find("CategoriesApp").exists()).toBe(true)
+    renderWithRelay()
+    expect(screen.getByText("The Art Genome Project")).toBeInTheDocument()
```

Comment:
The RTL way is to eschew internal implementation details like the name of the React component in favor of user-visible text.